cdi/4.0/_index.md

```diff
@@ -1,11 +1,31 @@
 ---
-title: "Jakarta Context Dependency Injection 4.0 (Under Development)"
-date: 2022-01-05
+title: "Jakarta Context Dependency Injection 4.0"
+date: 2022-02-22
 summary: "Release for Jakarta EE 10"
 ---
 
 Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
+* [Jakarta Contexts Dependency Injection 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.cdi/releases/4.0)
+  * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Contexts Dependency Injection 4.0 Specification Document](jakarta-cdi-spec-4.0.pdf) (PDF)
+* [Jakarta Contexts Dependency Injection 4.0 Specification Document](jakarta-cdi-spec-4.0.html) (HTML)
+* [Jakarta Contexts Dependency Injection 4.0 Javadoc](./apidocs)
+* [Jakarta Contexts Dependency Injection 4.0 TCK](https://download.eclipse.org/ee4j/cdi/4.0/cdi-tck-4.0.0-dist.zip)
+([sig](TBD_post_ballot),
+[sha](7671d6895eb57b74b52e46b63adfeb57adf965dd91efc673db21a781fedc452f),
+[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+
+* Maven coordinates
+  * [jakarta.enterprise:jakarta.cdi-api:jar:4.0.0](https://search.maven.org/artifact/jakarta.enterprise/jakarta.enterprise.cdi-api/4.0.0/jar)
+
+* Schemas
+  * [XML Schema for the CDI 4.0 beans.xml deployment](https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd)
+
+# Compatible Implementations
+
+* [Weld 5.0.0.CR1](https://weld.cdi-spec.org/download/)
+=======
```

Comment:
This seems not rendered. I think you need a space after the line 28.

Reply:
Ok, should be fixed.